strawberryfields/parameters.py

```diff
@@ -257,6 +258,8 @@ def par_convert(args, prog):
     """
 
     def do_convert(a):
+        if isinstance(a, blackbird.RegRefTransform):
+            a = a.expr
         if isinstance(a, sympy.Basic):
```

Comment:
```suggestion
        elif isinstance(a, sympy.Basic):
```

Reply:
This actually needs to stay as an `if` since if `a` is a `blackbird.RegRefTransform` then it'll be changed into a sympy expression, and thus also enter the next if (since `a` is then a `sympy.Basic`).

Reply:
Oh completely fair point!